auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -165,7 +167,8 @@ export default defineComponent({
   components: { EntityDetails, BaseVDataTable },
   props: {
     selectedColumns: { default: [] as string[] },
-    loading: { default: false }
+    loading: { default: false },
+    highlightIndex: { type: Number }
```

Comment:
I think should be `number` (type) not `Number` (class).

Reply:
Using `number` will cause an error here.
![image](https://github.com/bcgov/sbc-auth/assets/60866283/c92d3a59-8ce4-4ae0-b28b-90af61a53d6c)

Reply:
OK, this is probably due to Composition API syntax. Now I know.

Would it better to default highlightIndex to something instead (maybe even NaN)?

Reply:
@chenhongjing Would it make more sense to assign a default number (or NaN)?

Reply:
Sorry, I missed this comment. I set it as -1 by default now.